### DISCUSSION TITLE
Publications Page: Corrections and Additions

### DISCUSSION THREAD

**Discussion by baperry2** [Pele-Suite/Pele-Suite.github.io#3] [Announcements]
We're starting a running list of publications that develop Pele capability or use Pele (#2). Use this space to recommend corrections or additions to the list.

**Comment by drummerdoc** [comment]
Need to add the following PeleLMeX papers:

- Thermodiffusively-unstable lean premixed hydrogen flames: Length scale effects and turbulent burning regimes, EF Hunt, AJ Aspden, Combustion and Flame 272, 113855
- Three-dimensional phenomenology of freely-propagating thermodiffusively-unstable lean premixed hydrogen flames, AJ Aspden, TL Howarth, EF Hunt, Proceedings of the Combustion Institute 40 (1-4), 105634
- Thermodiffusively-unstable lean premixed hydrogen flames: Phenomenology, empirical modelling, and thermal leading points, TL Howarth, EF Hunt, AJ Aspden, Combustion and Flame 253, 112811
- Ensemble-averaged kinematics of harmonically oscillating turbulent premixed flames, K Kim, YJ Kim, AJ Aspden, D Shin, Combustion and Flame 253, 112815
- An empirical characteristic scaling model for freely-propagating lean premixed hydrogen flames, TL Howarth, AJ Aspden, Combustion and Flame 237, 111805
